Targets/duktape/README.md

```diff
@@ -0,0 +1,8 @@
+# Target: duktape
+
+To build duktape for fuzzing:
+
+1. Download the master branch of https://github.com/svaarala/duktape.
```

Comment:
Nit: please remove the '.' at the end of the two list items to match the style of the other README's